roxie/ccd/ccdserver.cpp

```diff
@@ -2228,6 +2236,7 @@ class CRoxieServerStrandedLateStartActivity : public CRoxieServerStrandedActivit
     {
         CRoxieServerStrandedActivity::reset();
         prefiltered = false;
+        eof = false;
```

Comment:
This feels like it's fixing a different bug (might mean lateStart activities in child queries misbehave?)